sormas-rest/src/main/java/de/symeda/sormas/rest/ImmunizationResource.java

```diff
@@ -143,4 +146,34 @@ public VaccinationDto createVaccination(
 		return FacadeProvider.getVaccinationFacade().getAllVaccinations(personUuid, disease);
 	}
 
+	@GET
+	@Path("/vaccination/{uuid}")
+	public VaccinationDto getVaccinationByUuid(@PathParam("uuid") String uuid) {
+		return FacadeProvider.getVaccinationFacade().getByUuid(uuid);
+	}
+
+	@POST
+	@Path("/editVaccination")
+	public VaccinationDto createVaccination(@Valid VaccinationDto vaccination) {
```

Comment:
Please align the naming of the endpoint as well as the method name with POST methods in other resources

Reply:
I made the changes, however there is a difference between the other endpoints.